packages/client-sdk-web/src/cache-client.ts

```diff
@@ -51,3 +54,20 @@ function createDataClient(props: CacheClientProps): IDataClient {
     defaultTtlSeconds: 60,
   });
 }
+
+function createPingClient(props: CacheClientProps): IPingClient {
+  // remove the 'cache.' portion of the data endpoint to get the ping URL
+  const cacheEndpoint = props.credentialProvider.getCacheEndpoint();
+  const pingUrl = cacheEndpoint.split('.').slice(1).join('.');
```

Comment:
🙈

Reply:
maybe we can add a `getPingEndpoint()` function to the creds provider?

Reply:
Considering that the [ping client constructor just reconstructs the cache endpoint](https://github.com/momentohq/client-sdk-nodejs/blob/d7b79455c4cd4844839309f878bea42abc9d6241/packages/client-sdk-web/src/internal/ping-client.ts#L34), I'd love to just pass the cache endpoint instead and just have the constructor add the `https://` and `443` bits. The fact that it wasn't already done this way made me think that might be problematic for some reason. If you don't have an objection to that, I'm gonna go for it.

Reply:
yes please